packages/chart/src/ChartTheme.ts

```diff
@@ -1,20 +1,52 @@
-import ChartTheme from './ChartTheme.module.scss';
+import { resolveCssVariablesInRecord } from '@deephaven/components';
+import Log from '@deephaven/log';
+import chartThemeRaw from './ChartTheme.module.scss';
 
-export default Object.freeze({
-  paper_bgcolor: ChartTheme['paper-bgcolor'],
-  plot_bgcolor: ChartTheme['plot-bgcolor'],
-  title_color: ChartTheme['title-color'],
-  colorway: ChartTheme.colorway,
-  gridcolor: ChartTheme.gridcolor,
-  linecolor: ChartTheme.linecolor,
-  zerolinecolor: ChartTheme.zerolinecolor,
-  activecolor: ChartTheme.activecolor,
-  rangebgcolor: ChartTheme.rangebgcolor,
-  area_color: ChartTheme['area-color'],
-  trend_color: ChartTheme['trend-color'],
-  line_color: ChartTheme['line-color'],
-  error_band_line_color: ChartTheme['error-band-line-color'],
-  error_band_fill_color: ChartTheme['error-band-fill-color'],
-  ohlc_increasing: ChartTheme['ohlc-increasing'],
-  ohlc_decreasing: ChartTheme['ohlc-decreasing'],
-});
+const log = Log.module('ChartTheme');
+
+export interface ChartTheme {
+  paper_bgcolor: string;
+  plot_bgcolor: string;
+  title_color: string;
+  colorway: string;
+  gridcolor: string;
+  linecolor: string;
+  zerolinecolor: string;
+  activecolor: string;
+  rangebgcolor: string;
+  area_color: string;
+  trend_color: string;
+  line_color: string;
+  error_band_line_color: string;
+  error_band_fill_color: string;
+  ohlc_increasing: string;
+  ohlc_decreasing: string;
+}
+
+export function initChartTheme(): Readonly<ChartTheme> {
```

Comment:
I'd probably name this `makeDefaultChartTheme()`, `getDefaultChartTheme()`, or `defaultChartTheme()`. The `init` makes me think it's initializing some sort of class or static property, when it's really just creating a new instance of the default theme.